codegen/smithy-aws-swift-codegen/src/main/resources/META-INF/services/software.amazon.smithy.swift.codegen.integration.SwiftIntegration

```diff
@@ -31,6 +31,3 @@ software.amazon.smithy.swift.codegen.protocols.rpcv2cbor.CborValidateResponseHea
 software.amazon.smithy.aws.swift.codegen.customization.rds.AuthTokenGeneratorIntegration
 software.amazon.smithy.aws.swift.codegen.customization.dsql.AuthTokenGeneratorIntegration
 software.amazon.smithy.aws.swift.codegen.customization.credentialresolverservices.InternalModelIntegration
-software.amazon.smithy.aws.swift.codegen.customization.credentialresolverservices.IdentityProvidingSTSClientIntegration
-software.amazon.smithy.aws.swift.codegen.customization.credentialresolverservices.IdentityProvidingSSOClientIntegration
-software.amazon.smithy.aws.swift.codegen.customization.credentialresolverservices.IdentityProvidingSSOOIDCClientIntegration
```

Comment:
These 3 integrations are eliminated since the files they generated are now static runtime files.